src/mimeparser.rs

```diff
@@ -390,23 +390,21 @@ impl MimeMessage {
                     prepend_subject = false
                 }
             }
-            if prepend_subject {
-                let subj = subject
-                    .find('[')
-                    .and_then(|n| subject.get(..n))
-                    .unwrap_or(subject)
-                    .trim();
```

Comment:
These lines removed everything before the first `[`, which resulted in subjects like `Re: ` (from `[deltachat/deltachat-core-rust] PR run failed`)